tox.ini

```diff
@@ -0,0 +1,106 @@
+[tox]
+envlist = py{39,310,311,312,313,314,py310}
+labels =
+    check = typing, gen_exports, type_completeness, pip_compile
+    cython = py39-cython2,py39-cython,py311-cython2,py313-cython
+
+# TODO:
+# * environment to check coverage
+# * replace ci.sh
+#   * --verbose --durations=10
+#   * -p _trio_check_attrs_aliases
+#   * mypy cache
+#   * LSP
+#   * apport
+# * use tox in CI
+# * switch to nox?
+# * move to pyproject.toml?
+#   * this means conditional deps need to be replaced
+
+# protip: install uv-tox for faster venv generation
+
+[testenv]
+description = "Base environment for running tests depending on python version."
```

Comment:
@jakkdl is configparser able to handle the quotes well? I think this is the first time I see them in this context.. Does the output of `tox l` look good?

Reply:
...no :sweat_smile: 
> ❯ tox l
default environments:
py39                 -> "Base environment for running tests depending on python version."
py310                -> "Base environment for running tests depending on python version."
py311                -> "Base environment for running tests depending on python version."
py312                -> "Base environment for running tests depending on python version."